### PR TITLE
bugherd task 633

### PR DESCRIPTION
**Pull request by rssag**
quote from the bugherd task: "Please make a hint regarding bootstrap user and credentials. Our customer often get stuck here already and this is often one of the first touchpoints with the documentation. The way it is described here it does not work"

ticket by marco.biegel@softwareag.com

not sure if this is sufficient? @sagscmi

### CODE REVIEW

File: content/device-integration/mqtt-bundle/integration-life-cycle.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ The cycle phase consists of two kinds of actions:
 
 ![MQTT phases](/images/mqtt/mqttDeviceIntegration.png)
 
+{{< c8y-admon-req >}}
+* Access via bootstrap user credentials
```
Comment (sagscmi) [inline content/device-integration/mqtt-bundle/integration-life-cycle.md:30]: Would it make sense to have a link to https://cumulocity.com/api/core/#tag/Device-credentials that explains how to get the device bootstrap credentials?

Comment (sagscmi) [inline content/device-integration/mqtt-bundle/integration-life-cycle.md:30]: Suggestion:
```suggestion
* Access via bootstrap user credentials.  See [Device credentials](https://cumulocity.com/api/core/#tag/Device-credentials) for more information on obtaining and using bootstrap credentials.
```

**Review by sagscmi — APPROVED** [review]
One optional improvement suggested.

File: content/device-integration/mqtt-bundle/integration-life-cycle.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ The cycle phase consists of two kinds of actions:
 
 ![MQTT phases](/images/mqtt/mqttDeviceIntegration.png)
 
+{{< c8y-admon-req >}}
+* Access via bootstrap user credentials. See [Device credentials](https://cumulocity.com/api/core/#tag/Device-credentials) for more information on obtaining and using bootstrap credentials.
```
Comment (MWindel) [inline content/device-integration/mqtt-bundle/integration-life-cycle.md:30]:
```suggestion
* Access via bootstrap user credentials. See [Device credentials](https://cumulocity.com/api/core/#tag/Device-credentials) for more information on how to obtain and use bootstrap credentials.
```

**Review by MWindel — APPROVED** [review]

**Pull request merged**